# HG changeset patch
# User joe
# Date 1542798089 25200
#      Wed Nov 21 04:01:29 2018 -0700
# Node ID ee4b3ea10f836fc228e383712594e866b38611b7
# Parent  7acfe551f5ad016133121b3c1876bf6d036fa53e
change window scale to 128MiB

My equipment can handle upto 1 Gbps. 128MiB at 100ms latency
will saturate it. Setting this to 16MiB to provide (hopefully)
128MiB/s at 100ms latency.

Be careful with this patch. On 9ramnode, it set the winsize=255 when
sending the ACK to betfair. That stops the betfair server from
responding any further as the 255 window size is too low.

diff -r 7acfe551f5ad -r ee4b3ea10f83 sys/src/9/ip/tcp.c
--- a/sys/src/9/ip/tcp.c	Wed Nov 21 02:08:15 2018 +0100
+++ b/sys/src/9/ip/tcp.c	Wed Nov 21 04:01:29 2018 -0700
@@ -86,8 +86,8 @@
 	 * these factors determine the ultimate bandwidth-delay product.
 	 * 64kb · 2⁵ = 2mb, or 2x overkill for 100mbps · 70ms.
 	 */
-	Maxqscale	= 4,		/* maximum queuing scale */
-	Defadvscale	= 4,		/* default advertisement */
+	Maxqscale	= 8 /* 4 */,		/* maximum queuing scale */
+	Defadvscale	= 8 /* 4 */,		/* default advertisement */
 };
 
 /* Must correspond to the enumeration above */
